bitnami/spark/README.md

```diff
@@ -0,0 +1,256 @@
+| `security.rpc.authenticationEnabled`        | Enable the RPC authentication                                                               | `no`                                                    |
+| `security.rpc.encryptionEnabled`            | Enable the encryption for RPC                                                               | `no`                                                    |
+| `security.storageEncryptionEnabled`         | Enable the encryption of the storage                                                        | `no`                                                    |
+| `security.ssl.enabled`                      | Enable the SSL configuration                                                                | `no`                                                    |
+| `security.ssl.needClientAuth`               | Enable the client authentication                                                            | `no`                                                    |
+| `security.ssl.protocol`                     | Set the SSL protocol                                                                        | `TLSv1.2`                                               |
+| `security.certificatesSecretName`           | Set the name of the secret that contains the certificates                                   | No default                                              |
+| `service.type`                              | Kubernetes Service type                                                                     | `ClusterIP`                                             |
+| `service.port`                              | spark client port                                                                           | `2379`                                                  |
+| `service.nodePort`                          | Port to bind to for NodePort service type (client port)                                     | `nil`                                                   |
+| `service.annotations`                       | Annotations for spark service                                                               | {}                                                      |
+| `service.loadBalancerIP`                    | loadBalancerIP if spark service type is `LoadBalancer`                                      | `nil`                                                   |
+| `ingress.enabled`                           | Enable the use of the ingress controller to access the web UI                               | `true`                                                  |
+| `ingress.hosts`                             | Add hosts to the ingress controller with name and path                                      | `name: spark.local`, `path: /`                          |
+
```

Comment:
I'm missing prometheus exporters. How viable is this? https://devrats.com/spark-prometheus/

Maybe, if this is something that cannot be generally applied to every spark job, what we could do is to allow adding custom sidecar containers (in the form of templates)

Reply:
The link you provide is not viable because it use node-exporter that is for physical nodes.
I found a workaround on this [link](https://medium.com/@salohyprivat/monitor-spark-streaming-with-prometheus-5b0eff5e318d) but not sure if it worth.

Reply:
Well, that's something we can take a look in future PRs

Reply:
Ok